src/github_client_wrapper.ts

```diff
@@ -1,16 +1,17 @@
-import { GitHub } from '@actions/github';
```

Comment:
this was no longer accessible, but `RestEndpointMethods` from octokit seemed to work